.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "Hacktoberfest"

```

Comment:
Opened https://github.com/streetsidesoftware/cspell-dicts/pull/596 so this hopefully won't be needed later on